README.rst

```diff
@@ -186,25 +186,27 @@ In addition to credentials, a number of other variables can be
-output      --output    output                AWS_DEFAULT_OUTPUT    Default output style
------------ ----------- --------------------- --------------------- ----------------------------
-ca_bundle   --ca-bundle ca_bundle             AWS_CA_BUNDLE         CA Certificate Bundle
------------ ----------- --------------------- --------------------- ----------------------------
-access_key              aws_access_key_id     AWS_ACCESS_KEY_ID     AWS Access Key
------------ ----------- --------------------- --------------------- ----------------------------
-secret_key              aws_secret_access_key AWS_SECRET_ACCESS_KEY AWS Secret Key
------------ ----------- --------------------- --------------------- ----------------------------
-token                   aws_session_token     AWS_SESSION_TOKEN     AWS Token (temp credentials)
-=========== =========== ===================== ===================== ============================
+================ =========== ===================== ===================== ============================
+Variable         Option      Config Entry          Environment Variable  Description
+================ =========== ===================== ===================== ============================
+profile          --profile   profile               AWS_DEFAULT_PROFILE   Default profile name
+---------------- ----------- --------------------- --------------------- ----------------------------
+region           --region    region                AWS_DEFAULT_REGION    Default AWS Region
+---------------- ----------- --------------------- --------------------- ----------------------------
+config_file                                        AWS_CONFIG_FILE       Alternate location of config
+---------------- ----------- --------------------- --------------------- ----------------------------
+output           --output    output                AWS_DEFAULT_OUTPUT    Default output style
+---------------- ----------- --------------------- --------------------- ----------------------------
+ca_bundle        --ca-bundle ca_bundle             AWS_CA_BUNDLE         CA Certificate Bundle
+---------------- ----------- --------------------- --------------------- ----------------------------
+access_key                   aws_access_key_id     AWS_ACCESS_KEY_ID     AWS Access Key
+---------------- ----------- --------------------- --------------------- ----------------------------
+secret_key                   aws_secret_access_key AWS_SECRET_ACCESS_KEY AWS Secret Key
+---------------- ----------- --------------------- --------------------- ----------------------------
+token                        aws_session_token     AWS_SESSION_TOKEN     AWS Token (temp credentials)
+---------------- ----------- --------------------- --------------------- ----------------------------
+timestamp_format             cli_timestamp_format                        Ouput format of timestamps
```

Comment:
Any reason this is ``timestamp_format`` here but ``cli_timestamp_format`` in the topic guide?